source/Calamari.Shared/Integration/Packages/Download/GitHubPackageDownloader.cs

```diff
@@ -193,7 +192,10 @@ static bool TryParseVersion(string input, out IVersion version)
                         client.DownloadFileWithProgress(uri, tempPath, (progress, total) => log.Progress(progress, $"Downloading {packageId} v{version}"));
 
                         DeNestContents(tempPath, localDownloadName);
-                        return PackagePhysicalFileMetadata.Build(localDownloadName);
+                        var packagePhysicalFileMetadata = PackagePhysicalFileMetadata.Build(localDownloadName);
+                        if (packagePhysicalFileMetadata == null)
+                            throw new CommandException($"Unable to retrieve metadata for package {packageId}, version {version}");
```

Comment:
Would this be cleaner?
```suggestion
                        var packagePhysicalFileMetadata = PackagePhysicalFileMetadata.Build(localDownloadName)
                        ?? throw new CommandException($"Unable to retrieve metadata for package {packageId}, version {version}");
```